.github/dependabot.yml

```diff
@@ -11,6 +16,18 @@ updates:
     directory: "/"
     schedule:
       interval: "weekly"
+    ignore:
+      - dependency-name: actions/cache
+      - dependency-name: actions/checkout
+      - dependency-name: actions/setup-python
+      # Managed by cisagov/skeleton-docker
+      # - dependency-name: actions/download-artifact
+      # - dependency-name: actions/github-script
+      # - dependency-name: actions/upload-artifact
+      # - dependency-name: docker/build-push-action
+      # - dependency-name: docker/login-action
+      # - dependency-name: docker/setup-buildx-action
+      # - dependency-name: docker/setup-qemu-action
```

Comment:
These lines should be uncommented.

Reply:
resolved with [d38e133](https://github.com/cisagov/domain-manager-ui/pull/247/commits/d38e1335f747335baa33f87d2745c9cf0cc84808)